week01/code/ArraysTester.cs

```diff
@@ -44,11 +44,11 @@ private static double[] MultiplesOf(double number, int length)
     
     /// <summary>
     /// Rotate the 'data' to the right by the 'amount'.  For example, if the data is 
-    /// <c>&lt;List&gt;{1, 2, 3, 4, 5, 6, 7, 8, 9}</c> and an amount is 3 then the list returned should be 
-    /// <c>&lt;List&gt;{7, 8, 9, 1, 2, 3, 4, 5, 6}</c>.  The value of amount will be in the range of <c>1</c> and 
-    /// <c>data.Count</c>.
-    /// <br /><br />
-    /// Because a list is dynamic, this function will modify the existing <c>data</c> list rather than returning a new list.
+    /// List{1, 2, 3, 4, 5, 6, 7, 8, 9} and an amount is 3 then the list after the function runs should be 
```

Comment:
Is the removal of the `<c>` tags and the &lt; &gt; entities intentional? I think it is, but just wanted to double check...

Reply:
Yeah, I don’t think they’re that helpful when shown in the code as comments. I suspect they were copied out of HTML at some point.